tools/snp.sh

```diff
@@ -95,7 +95,7 @@ AMDSEV_URL="https://github.com/ryansavino/AMDSEV.git"
 AMDSEV_DEFAULT_BRANCH="snp-latest-fixes"
 AMDSEV_NON_UPM_BRANCH="snp-non-upm"
 SNPGUEST_URL="https://github.com/virtee/snpguest.git"
-SNPGUEST_BRANCH="tags/v0.3.2"
+SNPGUEST_BRANCH="tags/v0.5.1"
```

Comment:
Please put this in a separate commit so that the rest of this can be cherry picked into the coco branch.